src/collections/components/CollectionForm/CollectionForm.tsx

```diff
@@ -108,9 +139,12 @@ export const CollectionForm: React.FC<
     // before they actually submit the form
     validateOnBlur: false,
     validateOnChange: false,
-    validationSchema: getValidationSchema(authorIds, languages),
+    validationSchema: getValidationSchema(authorIds),
     onSubmit: (values, formikHelpers) => {
-      onSubmit(values, formikHelpers);
+      // TODO: follow up with a separate PR/ticket that will be solely
+      // responsible for validating the labels before submission
+      // as they don't go through our normal form validation
+      onSubmit(values, formikHelpers, selectedLabels);
     },
```

Comment:
Filed ticket # 1684 (not linking it here for fear of JIRA automation automatically marking it done when this PR is merged) for the follow-up work.

Reply:
The tradoff for this ~hack~ unconventional approach of circumventing our form validation is that if we use the Autocomplete component instead of a temporary checkbox for the experimental label, we get support for multiple labels out of the box.